File: android_mobile/src/main/java/com/alexstyl/specialdates/facebook/FacebookImagePath.java
package com.alexstyl.specialdates.facebook;

import java.net.URI;

public final class FacebookImagePath {

    private static final int SIZE = 700;
    private static final String IMG_URL = "https://graph.facebook.com/%s/picture?width=" + SIZE + "&height=" + SIZE;

    private FacebookImagePath() {
        // hide this
    }

    public static URI forUid(long uid) {
        return URI.create(String.format(IMG_URL, uid));
    }
}

File: memento/src/main/java/com/alexstyl/specialdates/facebook/FacebookImagePath.kt
package com.alexstyl.specialdates.facebook

import com.alexstyl.specialdates.contact.ImageURL

object FacebookImagePath {

    private const val SIZE = 700
    private const val IMG_URL = "https://graph.facebook.com/%s/picture?width=$SIZE&height=$SIZE"

    fun forUid(uid: Long): ImageURL {
        return String.format(IMG_URL, uid)
    }
}
